Tests/js/views/assets/ez-searchview-tests.js

```diff
@@ -0,0 +1,193 @@
+            var searchRequestFired = false,
+                userSearchString = "AnySearch",
+                limit = 10;
+
+            this.view.on('*:searchRequest', function(e) {
+                searchRequestFired = true;
+                Assert.areSame(
+                    e.searchString, userSearchString,
+                    "the searchRequest should have the userSearchString in param"
+                );
+                Assert.areSame(
+                    e.limit, limit,
+                    "the searchRequest should have the limit in param"
+                );
+            });
+            this.view.set('loadMoreAddingNumber', limit);
+            this.view.render();
+
+            this.view.set('userSearchString', 'AnySearch');
+            Assert.isTrue(
+                searchRequestFired,
+                "searchRequest should have been fired after userSearchStringChange"
+            );
+        },
+
+        "Should set the limit and fire a search request": function () {
+            var searchRequestFired = false,
+                searchString = "AnySearch",
+                limit = 10,
+                loadMoreAddingNumber = 42,
+                that = this;
+
+            this.view.on('*:searchRequest', function(e) {
+                searchRequestFired = true;
+                Assert.areSame(
+                    e.searchString, searchString,
+                    "the searchRequest should have the userSearchString in param"
+                );
+                Assert.areSame(
+                    e.limit, that.view.get('limit'),
+                    "the searchRequest should have the limit in param"
+                );
+            });
+            this.view.set('searchString', 'AnySearch');
+
+            this.view.set('limit', limit);
+            this.view.set('loadMoreAddingNumber', loadMoreAddingNumber);
+
+            this.view.render();
+
+            this.view.get('searchListView').set('offset', '12');
+            this.view.fire('searchListView:offsetChange');
+
+            Assert.isTrue(
+                searchRequestFired,
+                "searchRequest should have been fired after offsetChange"
+            );
+            Assert.areSame(
+                that.view.get('limit'), limit + loadMoreAddingNumber,
+                "The limit should be updated"
+            );
+        },
+
+        "Should set the userSearchString after submitting form": function () {
+            var container = this.view.get('container');
+
+            this.view.render();
+
+            container.one('.ez-search-form-input').set('value', 'SomeResearch');
+            container.one('form').simulate('submit');
+            Assert.areSame(
+                this.view.get('userSearchString'),
+                'SomeResearch',
+                'Submitting form should set userSearchString'
+            );
+        },
+    });
+
+    attributesTest = new Y.Test.Case({
+        name: "eZ Search view attribute tests",
```

Comment:
"attribute tests" ? you should be a bit more precise, you are only testing `searchListView` attribute